ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -7,6 +7,10 @@ base_images:
     name: golangci-lint
     namespace: ci
     tag: v1.46.0
+  openstack-operator-ci-tempest:
```

Comment:
Already sets the name of container that is expected in tempest step[1]

[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-ref.yaml#L3